.github/workflows/test.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Cache nox
         uses: actions/cache@v4
         with:
-          key: test-${{ hashFiles('pyproject.toml') }}
+          key: test-${{ hashFiles('pyproject.toml') }}-${{ env.pythonLocation }}
```

Comment:
See https://github.com/wntrblm/nox/issues/735#issuecomment-1854599576

Reply:
This is weird. I have never encountered this bug before.